_bioinfcomputing/software_overview.md

```diff
@@ -0,0 +1,61 @@
+- Con: need to transfer files to Hutch servers, requires Internet connection, can be temporarily slow if Rhino has many concurrent users
+
+#### Running remotely on reserved cluster node (`Gizmo`)
+- Pro: higher CPU, memory and disk resources, and you're the exclusive user
+- Con: need to transfer files to Hutch servers, requires Internet connection, if you request a very powerful computer, you may have to wait a while for one to become available
+
+When using the Fred Hutch computing clusters, users should access these programming languages via the environmental modules (eg. `ml R` rather than simply `R` in Rhino). Doing this will ensure reproducibility of your code and that you're using the latest software available. More information about environmental modules are available [here](/bioinfcomputing/compute_environments/).
+
+
+## [Wiki Examples Repository](https://github.com/FredHutch/wiki-code-examples)
+
+In addition to the documentation provided in this wiki, we maintain a repository of coding examples to demonstrate the use of our coding standards, while showcasing some features and packages that could be useful in your research. We aim to include projects of various scope, so we can exhibit the software development principals in different types of user interfaces (e.g.: packages and interactive notebooks will both be included with appropriate documentation). 
+
+We encourage a broad range of coding example submissions from researchers, with the hope that it will enhance innovation and learning within Fred Hutch. The submission requirements are outlined in the following section, [Fred Hutch Software Development Standards](#fred-hutch-software-development-standards). In addition to creating reference material for your peers and gaining local notoriety as an awesome developer/researcher, you will have the added benefit of additional technical review and sharing/communicating your work with other researchers that may be interested in follow up.
+
+>Note: A common use of the examples repository will be to clone a copy locally and play with the available code. For this reason, we ask that data files not be stored in the repository.
+
+
+### Accessing the Examples
+
+Examples can be explored in a web browser by clicking the link above. Alternatively, you can use github to clone a copy to your local machine, where you can modify and play with the example code independently. 
+
+```
+git clone https://github.com/FredHutch/wiki-code-examples.git
+```
+
+For more information on using Github, see the [Managing and Sharing Code](/bioinfcomputing/software_managecode/) page in this wiki.
+
+
+### Submitting Your Code
+When you have completed a coding project that you would like to submit to the examples wiki, there are a few steps in Github to navigate the review process. You will want to push your code in a clean branch to the repository, and then submit a pull request on `github.com` for your branch that will kick off a review and merge process.
```

Comment:
I personally prefer instructions for submission to be kept within the repo itself. If we do want to keep this here, there should be a description to how to get added to the GitHub org so it's possible to create a branch within the original repo.